package-lock.json

```diff
@@ -0,0 +1,1082 @@
+{
```

Comment:
pls remove this file, we use yarn here :)

Reply:
Sorry, haven't noticed. Done!